deploy/deploy.ps1

```diff
@@ -45,17 +46,22 @@ while(($selection -ne 1) -and ($selection -ne 2))
 
 if($selection -eq 1) {
     $agMode = $true
-    $message = "Deploying in AVAILABILITY GROUP mode...`n
-Please get your AG Name ready..."
+    $message = "Deploying in AVAILABILITY GROUP mode...`n"
+    if($agName -eq $null){
+        $message+="Please get your AG Name ready..."
+    }else{
+        $message+="...to AG Name: [$agName]."
+    }
 }else{
     $agMode = $false
+    $agName = "x"

```

Comment:
x is an odd magic string, I see it in a few other places. I think its supposed to be Dummy_PlaceHolder, and having something long makes searching for all uses of 'magic string' easier (vs. search for x).

Reply:
Good call, changing the magic string also makes the "Press X to abort" message less ambiguous in context.